zeroconf/__init__.py

```diff
@@ -509,7 +509,7 @@ def write(self, out: 'DNSOutgoing') -> None:
 
     def to_string(self, other: Union[bytes, str]) -> str:
         """String representation with additional information"""
-        arg = "%s/%s,%s" % (self.ttl, self.get_remaining_ttl(current_time_millis()), cast(Any, other))
+        arg = "%s/%s,%s" % (self.ttl, int(self.get_remaining_ttl(current_time_millis())), cast(Any, other))
```

Comment:
See my comment above. Either a separate pull request or this one should be renamed.